README.md

```diff
@@ -1 +1,176 @@
-# First_Capstone_Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="assets/celec.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h2><b>Capstone Project</b></h2>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
```

Comment:
- [OPTIONAL] Since you built this project solely by yourself, I think you should correct this to `[👥 Author](#author)`, without the "s", and change it also in the author content header, as in `## 👥 Author <a name="author"></a>`